stripecardscan/build.gradle

```diff
@@ -22,7 +22,9 @@ dependencies {
     api project(":stripe-core")
     implementation project(":camera-core")
 
-    implementation "org.tensorflow:tensorflow-lite:2.8.0"
+    // Allow the user to specify their own version of Tensorflow Lite to include
+    runtimeOnly project(":stripecardscan-tflite")
```

Comment:
I would guess since we have the smaller version, no user would want to use the vanilla tensorflow lite dep? If that's the case instead of creating a new module, is it possible to just save the aar file inside stripecardscan and link it directly?

Reply:
It's possible that a merchant is already using TFLite, and it would interfere if there are two definitions of TFLite